File: modul 4/soal 5/PRAK305-2210817220034-Wanda Karliyanti.c
#include<stdio.h>

main(){
    for(int z=1;z<=5;z++){
int hari, jam, menit, detik, d;
scanf("%d", &d);

hari=d%(86400*30)/86400;
jam=(d%86400)/3600;
menit=d%3600/60;
detik=d%60;

if(hari>=1){
    printf("%d hari %.2d:%.2d:%.2d", hari, jam, menit, detik);}

else{printf("%.2d:%.2d:%.2d\n", jam, menit, detik);}

}
}

File: modul 4/soal 5/PRAK305-2210817220034-Wanda Karliyanti.py
for z in range(5):
    d=int(input())
    hari=d%(86400*30)/86400
    jam=(d%86400)/3600
    menit=d%3600/60
    detik=d%60
    if(hari>=1):
        print("%d hari %.2d:%.2d:%.2d" %(hari,jam,menit,detik))
    else:
        print("%.2d:%.2d:%.2d\n" %(jam,menit,detik))